sdk/communication/azure-communication-networktraversal/pom.xml

```diff
@@ -124,6 +124,11 @@
       <version>1.1.4</version> <!-- {x-version-update;com.azure:azure-communication-identity;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure.tools</groupId>
+      <artifactId>azure-autorest-customization</artifactId>
+      <version>1.0.0-beta.4</version>
+    </dependency>
```

Comment:
This should not be added in this pom.xml. You might want to create a separate pom.xml under `swagger` directory and then add your customization code. See https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/search/azure-search-documents/swagger for example.